driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedTest.java

```diff
@@ -156,32 +164,47 @@ public Entities getEntities() {
     }
 
     @NonNull
-    protected static Collection<Arguments> getTestData(final String directory) throws URISyntaxException, IOException {
```

Comment:
This should be a straightforward inline, along with the changes to the test parameters.